src/twisted/web/test/test_cgi.py

```diff
@@ -119,7 +120,9 @@ def testCGI(self):
         cgiFilename = self.writeCGI(DUMMY_CGI)
 
         portnum = self.startServer(cgiFilename)
-        d = client.getPage("http://localhost:%d/cgi" % portnum)
+        d = client.Agent(reactor).request(
+            'GET', "http://localhost:%d/cgi" % (portnum,))
```

Comment:
(optional) it might be more consistent to use `'` or `"` instead of mixing the two. But, since the linter doesn't care, I probably shouldn't either :)